src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -325,6 +354,10 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Equal("System.Int32", model.GetTypeInfo(def).ConvertedType.ToTestDisplayString());
             Assert.Null(model.GetSymbolInfo(def).Symbol);
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
>Assert.Equal(0, model.GetConstantValue(def).Value); [](start = 12, length = 51)

What type is returned by SemanticModel for the "def" node here? That type and the type of the constant value should agree.